notify/notify.go

```diff
@@ -785,6 +797,11 @@ func NewTimeMuteStage(mt map[string][]timeinterval.TimeInterval) *TimeMuteStage
 // Exec implements the stage interface for TimeMuteStage.
 // TimeMuteStage is responsible for muting alerts whose route is not in an active time.
 func (tms TimeMuteStage) Exec(ctx context.Context, l log.Logger, alerts ...*types.Alert) (context.Context, []*types.Alert, error) {
```

Comment:
it might be more convenient to have a generic `TimeStage` that could work for both active and mute depending on a flag? And the pipeline would run the active interval stage before the mute interval stage.

Reply:
I see it make sense, thanks